lib/WasmReader/WasmBinaryReader.cpp

```diff
@@ -442,7 +501,7 @@ WasmBinaryReader::ResetModuleData()
     m_visitedSections = BVFixed::New<ArenaAllocator>(bSectLimit + 1, &m_alloc);
```

Comment:
It is to validate precedence.
Because although the order is fixed, the presence of a section is not mandatory.
Therefor, you could have section `function_bodies` but missing `function_signatures`